src/crewai/cli/cli.py

```diff
@@ -179,5 +183,64 @@ def run():
     run_crew()
 
 
+@crewai.command()
+def signup():
```

Comment:
It's not that clear that you should call `signup` in order to login. Should we have a dedicated command or an alias?

Reply:
Yeah, I'll add the same function but with the def login just in case